synfig-core/src/modules/lyr_freetype/lyr_freetype.cpp

```diff
@@ -705,12 +705,12 @@ Layer_Freetype::get_param_vocab(void)const
 
 	ret.push_back(ParamDesc("use_kerning")
 		.set_local_name(_("Kerning"))
-		.set_description(_("Enables/Disables font kerning (If the font supports it)"))
+		.set_description(_("When checked. enables font kerning (If the font supports it)"))
```

Comment:
Looks like there should be a comma here?

Reply:
Right, nice catch!